dspace/config/registries/dublin-core-types.xml

```diff
@@ -544,6 +734,14 @@
     <scope_note>MEdical Subject Headings</scope_note>
   </dc-type>
 
+  <!-- specific type required by OpenAIRE4 Guidelines -->
+  <dc-type>
+    <schema>dc</schema>
+    <element>subject</element>
+    <qualifier>fos</qualifier>
+    <scope_note>Fields of Science and Technology - OECD</scope_note>
```

Comment:
This seems like it might be more appropriate in the `oaire` schema, as  I'm not sure it'll be used outside of the OpenAIRE context.

Reply:
We move it to oaire, but keeping the same schema. Because it makes sense in that context.